pyproject.toml

```diff
@@ -1,11 +1,12 @@
 [tool.poetry]
 name = "PyArtifactory"
-version = "2.9.0"
+version = "2.10.0"
 description = "Typed interactions with the Jfrog Artifactory REST API"
 authors = [
     "Ananias CARVALHO <carvalhoananias@hotmail.com>",
     "Thomas GAUDIN <thomas.gaudin@centraliens-lille.org>",
-    "Helio Chissini de Castro <heliocastro@gmail.com"
+    "Helio Chissini de Castro <heliocastro@gmail.com",
+    "Eero Aaltonen <eero.aaltonen@vaisala.com>"
```

Comment:
No need to update this file, I habitually create a separate MR for upgrading the lib version. Also, I do not think this is necessary to update the author's list since you will automatically be added as contributor when the MR is merged.